programs/psinode/main.cpp

```diff
@@ -2374,8 +2374,48 @@ void run(const std::string&              db_path,
          {
             if (!showedBootMsg)
             {
-               PSIBASE_LOG(node.chain().getLogger(), notice)
-                   << "Need genesis block; use 'psibase boot' to boot chain";
+               std::string xAdminSubdomain;
+               for (const auto& service : services)
+               {
+                  if (service.root.string().find("services/x-admin") != std::string::npos &&
+                      service.host.ends_with('.'))
+                  {
+                     xAdminSubdomain = service.host + host;
+                     break;
+                  }
+               }
+
+               if (!xAdminSubdomain.empty())
+               {
+                  std::string protocol;
+                  std::string port;
+                  for (const auto& listen : http_config->listen)
+                  {
+                     if (std::holds_alternative<psibase::http::tcp_listen_spec<true>>(listen))
+                     {
+                        const auto& spec = std::get<psibase::http::tcp_listen_spec<true>>(listen);
+                        protocol         = "https://";
+                        port             = std::to_string(spec.endpoint.port());
+                        break;
+                     }
+                     else if (std::holds_alternative<psibase::http::tcp_listen_spec<false>>(listen))
+                     {
+                        const auto& spec = std::get<psibase::http::tcp_listen_spec<false>>(listen);
+                        protocol         = "http://";
+                        port             = std::to_string(spec.endpoint.port());
+                     }
+                  }
+                  xAdminSubdomain = protocol + xAdminSubdomain + ":" + port;
+               }
+
+               std::string message = "Node is not yet booted. To boot, use the 'psibase boot' CLI";
```

Comment:
I think this should try to use the same terminology as the web UI.

Reply:
Are you referring to "Need genesis block;" != "Node is not yet booted."?

Reply:
I propose to change all instances of:
"Need genesis block; use 'psibase boot' to boot chain"
to
"Node is not yet booted. To boot, use the 'psibase boot' CLI"

And then also leave the extra special output exclusively in the psinode CLI. I want the "homepage" of an unbooted node to eventually redirect to x-admin.

Reply:
I was referring to the whole thing. The setup page has "Join Network" / "Create Network".

Reply:
I was *not* referring to what the API endpoints return for an unbooted chain.

Reply:
Updated the message as discussed